config/configuration.json

```diff
@@ -1290,6 +1308,9 @@
       "defaultPriority": "LOW"
     }
   },
+  "ElevationUtilities": {
+     "elevation.path": "extra/elevation"
+  },
```

Comment:
I see that you removed elevation.path in the ElevationUtilities md file... but added it here. Should this be elevation.prefix?